airflow/models.py

```diff
@@ -1446,6 +1446,7 @@ def signal_handler(signum, frame):
 
                 Stats.incr('operator_successes_{}'.format(
                     self.task.__class__.__name__), 1, 1)
+                Stats.incr('task_success')
```

Comment:
success->successes (and same with failure) to be consistent with the other metric. Even better ti_successes since tasks are abstract.